libraries/apollo-ast/api/apollo-ast.api

```diff
@@ -846,8 +846,9 @@ public abstract interface class com/apollographql/apollo3/ast/NodeTransformer {
 public final class com/apollographql/apollo3/ast/ParserOptions {
 	public static final field Companion Lcom/apollographql/apollo3/ast/ParserOptions$Companion;
 	public fun <init> ()V
-	public fun <init> (ZZZ)V
-	public synthetic fun <init> (ZZZILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun <init> (ZZZZ)V
+	public synthetic fun <init> (ZZZZILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public final fun getAllowClientControlledNullability ()Z
```

Comment:
Might want to turn that into a Builder now that I think about it 🤔 . The more I do APIs, the more I like Builders...

Reply:
Builder is in the place